site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -40,12 +40,22 @@ of type [Ruleset][4], as shown below.
 ## Rule Resources
 
 An extension can specify up to [MAX_NUMBER_OF_STATIC_RULESETS][5] [rulesets][6] as part of the
-`"rule_resources"` manifest key. The number of rules across **enabled** static rulesets must be less
-than the [MAX_NUMBER_OF_RULES][7] constant.
+`"rule_resources"` manifest key. An extension is allowed to enable up to at least
```

Comment:
What does "up to at least" mean? I read "up to" as an upper limit, and "at least" as a lower limit...

Reply:
This is more of a lower limit. so I'll use "at least" here